writer/writer.py

```diff
@@ -2,4 +2,7 @@
 
 
 def write(cmd: egg_info, basename: str, filename: str):
+    should_write = cmd.distribution.name == 'writing'
```

Comment:
I'm finding name is sometimes ``0``, when running ``setup.py egg_info``.
Instead, I've used ``cmd.name``, which comes from https://github.com/pypa/setuptools/blob/v51.3.3/setuptools/command/egg_info.py#L123. Reasonable?

Reply:
Yes, that seems reasonable.

```suggestion
    should_write = cmd.name == 'writing'
```